packages/flutter/test/material/checkbox_test.dart

```diff
@@ -1624,7 +1964,7 @@ void main() {
     expect(find.text(tapTooltip), findsOneWidget);
   });
 
-  testWidgets('Checkbox has default error color when isError is set to true - M3', (WidgetTester tester) async {
+  testWidgets('Checkbox has default error color when isError is set to true - M3 only', (WidgetTester tester) async {
```

Comment:
There is `Material3` in other tests but here its `M3` for the "only" test. Is this intentional?